nextjs-blog/components/cardimage.js

```diff
@@ -1,26 +1,49 @@
-import { useRef, useEffect } from "react";
+import { useRef, useEffect, useState } from "react";
 import { register } from "swiper/element/bundle";
-import Image from "next/image";
-import styles from "../styles/swipercontainer.module.scss";
+import Image from "next/legacy/image";
+import dbMocks from "../__mocks__/dbMocks.json";
 
 register();
 
-export default function CardImage({ imageUrl }) {
+export default function CardImage({
+  currentevent,
```

Comment:
V minor, but convention is to camelCase

`currentevent -> currentEvent`
`setCurrentevent -> setCurrentEvent`

Reply:
Camel Case Corrected!